volplugin/handlers.go

```diff
@@ -168,6 +173,9 @@ func mount(master, host string) func(http.ResponseWriter, *http.Request) {
 			return
 		}
 
+		stopChan := addStopChan(uc.Request.Name)
+		go heartbeatMount(master, ttl, ut, stopChan)
```

Comment:
So, this is an interesting first-responder problem. Basically, if we don't acquire the lock first, someone else can try to mount it and succeed. If we acquire the lock first, the next attempt to mount will break allowing the functionality we want.

Basically this defeats a mount race across the cluster. I don't believe there's a better way to do it, but I'm open to suggestions.